packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/InviteCollaborator/AutocompleteItem.spec.js

```diff
@@ -25,7 +25,13 @@ describe('AutocompleteItem component', () => {
       expect(wrapper.find('oc-icon-stub').attributes().name).toEqual('group')
       expect(wrapper.find('oc-icon-stub').rootNode.key).toEqual('avatar-group')
     })
-    it.each([3, 4, 6])(
+    it('should display a guest icon for guest shares', () => {
+      const wrapper = createWrapper({ shareType: 4 })
```

Comment:
I know that ints were used before, but please use the respective `ShareType.xyz.value` here and in line 34 instead of magic numbers.